packages/bpk-scrim-utils/src/withScrimmedPortal-test.tsx

```diff
@@ -26,27 +26,27 @@ import type { Props } from './withScrimmedPortal';
 
 describe('withScrimmedPortal', () => {
   it('renders the wrapped component inside a portal correctly with fallback to document.body', () => {
-    const DialogContent = () => <div>Dialog content</div>;
+    const DialogContent = () => <div data-testid="dialog-content">Dialog content</div>;
     const ScrimmedComponent = withScrimmedPortal(DialogContent);
 
     render(
       <div id="pagewrap">
-        <div> Content hidden from AT</div>
+        <div data-testid="hidden"> Content hidden from AT</div>
         <ScrimmedComponent
           getApplicationElement={() => document.getElementById('pagewrap')}
         />
       </div>
     );
-    expect(document.body).toMatchSnapshot();
+    expect(document.body).toContainElement(screen.getByTestId('dialog-content'));
   });
 
   it('renders the wrapped component inside a portal with renderTarget provided', () => {
-    const WrappedComponent = () => <div>Wrapped Component</div>;
-    const ScrimmedComponent = withScrimmedPortal(WrappedComponent);
+    const DialogContent = () => <div data-testid="dialog-content">Dialog content</div>;
```

Comment:
what's the difference between this test and the next one?

Reply:
hmm, yeah, you're right. Nothing really, I'll remove it.